ark/utils/spatial_analysis_utils_test.py

```diff
@@ -142,16 +149,14 @@ def test_get_pos_cell_labels_channel():
 
 def test_get_pos_cell_labels_cluster():
     all_data, _ = make_example_data_closenum()
-    example_thresholds = make_threshold_mat()
 
     # Only include the columns of markers
-    fov_channel_data = all_data.drop(all_data.columns[[
-        0, 1, 14, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32]], axis=1)
+    all_data.drop(all_data.columns[[0, 1, 14, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32]], axis=1)
```

Comment:
Can we use PRE_CHANNEL and POST_CHANNEL here?

Reply:
yep.  missed that one.